src/modules/auth/components/Form/RegisterForm/RegisterForm.js

```diff
@@ -0,0 +1,46 @@
+import * as yup from "yup";
```

Comment:
I already told you I don't see any sense in taking it out of the `RegisterForm` component to the other file. Merge them and declare them before the component code

Reply:
forgot to remove it. I moved id to .jsx but forgot remove .js